src/macros.jl

```diff
@@ -237,7 +246,16 @@ function parse_constraint_head(_error::Function, ::Val, args...)
     _unknown_constraint_expr(_error)
 end
 function parse_constraint(_error::Function, args...)
-    _unknown_constraint_expr(_error)
+    # Define this as the last fallback: either this is a function call that may
+    # be overridden by extensions, or a syntax that is not recognised.
+    # Multiple dispatch does not work here, due to ambiguity with:
+    #     parse_constraint(_error::Function, lb, lsign::Symbol, aff, rsign::Symbol, ub)
+    if args[1] isa Symbol
+        (sense, vectorized) = _check_vectorized(args[1])
+        vectorized, parse_one_operator_constraint(_error, vectorized, Val(sense), args[2:end]...)...
```

Comment:
I guess that
```julia
julia> @constraint(model, g(1))
ERROR: LoadError: In `@constraint(model, g(1))`: Constraints must be in one of the following forms:
       expr1 <= expr2
       expr1 >= expr2
       expr1 == expr2
       lb <= expr <= ub
Stacktrace:
 [1] error(::String, ::String) at ./error.jl:42
 [2] _macro_error(::Symbol, ::Array{Any,1}, ::String) at /home/blegat/.julia/packages/JuMP/MnJQc/src/macros.jl:856
 [3] (::JuMP.var"#_error#68"{Symbol})(::String) at /home/blegat/.julia/packages/JuMP/MnJQc/src/macros.jl:331
 [4] parse_constraint(::JuMP.var"#_error#68"{Symbol}, ::Symbol, ::Int64) at /home/blegat/.julia/packages/JuMP/MnJQc/src/macros.jl:220
 [5] _constraint_macro(::Tuple{Symbol,Expr}, ::Symbol, ::Function) at /home/blegat/.julia/packages/JuMP/MnJQc/src/macros.jl:372
 [6] @constraint(::LineNumberNode, ::Module, ::Vararg{Any,N} where N) at /home/blegat/.julia/packages/JuMP/MnJQc/src/macros.jl:455
in expression starting at REPL[4]:1
```
now throws a MethodError.
Can you as it as a test, and add a fallback method for `parse_one_operator_constraint` that calls `_unknown_constraint_expr` ?

Reply:
Is the new commit good enough?